webapp/store/survey/surveyInfo/reducer.js

```diff
@@ -5,11 +5,6 @@ import * as Survey from '@core/survey/survey'
 // App actions
 import { SystemActions } from '@webapp/store/system'
 
-// Processing chain actions
-import { chainSave, chainDelete } from '@webapp/loggedin/modules/analysis/chain/actions'
-import { stepDelete } from '@webapp/loggedin/modules/analysis/step/actions'
-import { calculationDelete } from '@webapp/loggedin/modules/analysis/calculation/actions'
-
 import * as SurveyActions from '../actions'
```

Comment:
when saving chain, do we set the survey as draft?
if not,  in saveChain action we should ```dispatch(SurveyActions.markSurveyDraft())```

Reply:
i have to fix that ahhaha

Reply:
this branch or another one?

Reply:
other branch